api/app/hfi/pdf_template.py

```diff
@@ -1,8 +1,16 @@
 """String representations of templates for in memory loading"""
+from enum import Enum
 import os
 
 CSS_PATH = os.path.join(os.path.dirname(__file__), "style.css")
 
+
+class PDFTemplateName(Enum):
+    """Enumerates the possible templates client can ask for"""
+    DAILY = 'daily'
+    PREP = 'prep'
+
+
 DAILY_TEMPLATE_HTML = """
```

Comment:
I'm not too keen on inline html - can't we load this from a file? Just read it in as a string.

Reply:
Was trying to avoid loading HTML files every call, but I think it's a decent trade off in favor of readability to do so

Reply:
It only happens on pdf rendering, it shouldn't make it too slow. If you're worried, you can always stick it int that singleton class.